src/Accordion/Accordion.css

```diff
@@ -0,0 +1,68 @@
+.accordion-wrapper {
+  width: 100%;
+  justify-content: center;
+}
+
+.accordion-item {
+  border: 2px solid #333;
+  border-radius: 4px;
+  margin: 5px 0;
+  padding: 2px 4px;
+}
+
+.accordion-title {
+  display: flex;
```

Comment:
- In css overall its good practice to have aggregated definitions of functions in order if applicable

1. display (none/hidden)
2. posiiton and underneath all of the values(inset, top, left...)
3. all of the definition for flex/grid if present
4. width/height
5. fonts
6. transforms/margins/paddings
7. transition/animations
8. z-index